sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -2160,3 +2160,7 @@ def test_as_real_imag():
 def test_issue_18746():
     e3 = cos(S.Pi*(x/4 + 1/4))
     assert e3.period() == 8
+
+
+def test_atan_inf_left():
```

Comment:
I do believe that we would like to see some sort of `PoleError` here but I would like to test out more cases (think of cases that could go wrong) . the `acot` function is related to the `atan` function hence there is a possiblity of cases going wrong there too.
```
>>> atan(x).rewrite(acot)
acot(1/x)
```

Also could you please change the name of the function (use `test_issue_issue_number` like other functions) and place it in sorted order